client/src/actions/books.js

```diff
@@ -0,0 +1,145 @@
+import axios from 'axios';
+import { SET_BOOKS, ADD_BOOK, BOOK_FETCHED, BOOK_UPDATED, BOOK_DELETED } from './types';
+import { addFlashMessage } from './flashMessages';
+
+const API_VERSION = window.API_VERSION;
+
+export function setBooks(books) { // eslint-disable-line require-jsdoc
+  return {
+    type: SET_BOOKS,
+    books
+  };
+}
+
+/**
+ * 
```

Comment:
You could delete lines 15 and 16 if there's no description for the function - and in all other occurrences in this patch.